DEPS

```diff
@@ -11,17 +11,25 @@ Core dependencies
 
 Essential:
 
-* python (== 2.7.x)
+* python3 (>= 3.6)
 * bsddb (distributed with stdlib) or bsddb3
 * gtk+ >= 3.10
 * gstreamer >= 1.4
 * gstreamer-plugins-base >= 1.6
 * gstreamer-plugins-good >= 1.4
-* mutagen (>= 1.10)
-* dbus-python
-* pygobject (>= 3.13.2)
-* python-cairo
+* python3-mutagen (>= 1.10)
 * GI typelib files for GTK+, GStreamer (including gstreamer-plugins-base) and cairo (python-gi-cairo on Ubuntu)
```

Comment:
You can remove the part about python-gi-cairo.

Reply:
Done.